dali/pipeline/operators/reader/video_reader_op.h

```diff
@@ -83,12 +102,19 @@ class VideoReader : public DataReader<GPUBackend, SequenceWrapper> {
                                   prefetched_sequence.sequence.raw_data(),
                                   prefetched_sequence.sequence.size(),
                                   ws->stream());
+
+        if (enable_file_root_) {
+          auto *label = label_output->template mutable_tensor<int>(data_idx);
```

Comment:
I think that you don't need a template  disambiguator, label_output type is known.